policy/custom/definitions/policyset/DefenderForCloud.bicep

```diff
@@ -9,15 +9,10 @@
 
 targetScope = 'managementGroup'
 
-@description('Management Group scope for the policy definition.')
-param policyDefinitionManagementGroupId string
-
-var customPolicyDefinitionMgScope = tenantResourceId('Microsoft.Management/managementGroups', policyDefinitionManagementGroupId)
-
 resource ascAzureDefender 'Microsoft.Authorization/policySetDefinitions@2020-03-01' = {
   name: 'custom-enable-azure-defender'
   properties: {
-    displayName: 'Custom - Azure Defender for Azure Services'
+    displayName: 'Custom - Microsoft Defender for Cloud'
```

Comment:
Should the wording "Custom - " be removed since this is a built-in policy now?
```suggestion
    displayName: 'Microsoft Defender for Cloud'
```

Reply:
I used the same policy names as listed in Azure Portal so there's alignment.  I think we should keep it as-is.

![image](https://user-images.githubusercontent.com/16688027/143321067-c121106a-5a47-4cd9-a355-2473bde938b4.png)

For "Custom" in the prefix, we should keep it since it's a custom policy initiative (policy set).

Reply:
Gotcha, thanks for the confirmation @SenthuranSivananthan. Approved from my end!